user-guide/Standard_Apps/EPM/EPM_VSAT/VSAT_deployment.md

```diff
@@ -4,15 +4,109 @@ uid: VSAT_deployment
+1. On the **DB Polling Config** page, adjust the polling parameters according to your preferences.
+
+1. On the **Collector Setup** page, configure the following settings:
+
+   - **File Import**: Enable *File Import* and *File Export* if CMDB (customer data) is available.
+   - **ID Import/Export**: Enable this feature and specify the paths to the CA and CR folder.
+   - **System Credentials**: Enter the system credentials.
+   - **Entity Removal File Path**: Specify the path for the entity removal file.
+
+1. Right-click the element and add the *NETWORK* custom property.
+
+   See [Adding a custom property](xref:Managing_element_properties#adding-a-custom-property-to-an-item).
+
+### Newtec Dialog collectors
+
+1. In DataMiner Cube, create the collector elements using the *Newtec Dialog Platform VSAT* connector.
+
+   When configuring the element connections, use the same HTTPS URL for all three required HTTP connections.
+
+1. On the **User Authentication** page of the new elements, enter the usernames and passwords.
+
+1. On the **Polling Config** page, set *TSDB Polling Status* to enabled, adjust the TSDB parameters as needed, and click *Auto Configure*.
+
+1. On the **Polling Status** page, adjust the polling parameters according to your preferences.
+
+1. On the **Collector Setup** page, configure the following settings:
+
+   - **File Import**: Enable *File Import* and *File Export* if CMDB (customer data) is available.
+   - **ID Import/Export**: Enable this feature and specify the paths to the CA and CR folder.
+   - **System Credentials**: Enter the system credentials.
+   - **Entity Removal File Path**: Specify the path for the entity removal file.
+
+1. Right-click the element and add the *NETWORK* custom property.
+
+## EPM configuration
+
+Follow the steps below to set up the **front-end and back-end elements**:
+
+1. In DataMiner Cube, create the front-end and back-end elements using the *Skyline EPM Platform VSAT GEO* connector.
+
+1. For each of the collectors, the front end, and the back-end elements, right-click the element, select *Properties*, and note down the element ID (in the format "DMAID/ElementID") displayed in the *Properties* window.
+
+   You will need these IDs later to fill them in in configuration tables.
+
+1. Configure the front-end element, using either the Visual page, which uses the Visual Overview assigned by the package you installed, or the data pages in the [Monitoring app](xref:Accessing_the_Monitoring_app):
+
+   - **Role**: Set the Role parameter to *Frontend*.
+   - **System Credentials and File Handling**: Configure system credentials and EPM file paths.
+   - **Backup and Tables**: Configure backup settings and populate tables with collector and backend IDs.
+
+1. Configure the back-end elements in a similar way:
+
+   - **Role**: Set the Role parameter to *Backend*.
+   - **File Handling**: EPM file paths must point to the FE server.
+   - **Collectors Table**: Include only local collector IDs.
```

Comment:
The instructions for the FE and BE configuration (line 75 - 85) are currently very vague. I think it would be better to pick one option, either Visual Overview or the Monitoring app, and then to clearly mention which parameters need to be adjusted there and on which page. Right now, users will need to go searching for them, and that is not user-friendly.